libbeat/cmd/instance/metrics_other.go

```diff
@@ -1,8 +1,11 @@
-// +build !darwin,!linux,!windows darwin,!cgo linux,!cgo windows,!cgo
+// +build darwin,!cgo freebsd,!cgo openbsd,!cgo
```

Comment:
Definitely double check my work, but I think we have a different set of built tags.

From metrics.go (without openbsd):
`(darwin AND cgo) OR (freebsd AND cgo) OR linux OR windows`

Negating this and simplifying it:
```
!((darwin AND cgo) OR (freebsd AND cgo) OR linux OR windows)
!(darwin AND cgo) AND !(freebsd AND !cgo) AND !linux AND !windows
(!darwin OR !cgo) AND (!freebsd OR !cgo) AND !linux AND !windows
```

Conversion to build tags:
```
// +build !darwin !cgo
// +build !freebsd !cgo
// +build !linux,!windows
```